### ISSUE TITLE
Block comment can result in incorrect minification

### ISSUE THREAD

**Issue by thdoan** [clean-css/clean-css#1238]
__IMPORTANT: clean-css is now in a [maintenance mode](https://github.com/clean-css/clean-css/discussions/1209). PRs are still welcome, and I will try do an occasional bugfix relase.__

### Precheck

* Do a quick search and make sure a bug has not yet been reported;
* do a quick check if the bug still exists in the latest patch version;
* finally, be nice and have fun!

### Environment

* clean-css version - `npm ls clean-css`: clean-css@5.3.1
* node.js version - `node -v`: v16.14.2
* operating system: Windows 10

### Configuration options

```js
// rollup_config.js
import CleanCSS from 'clean-css';
...
transform: (contents) => new CleanCSS().minify(contents).styles,
```

### Input CSS

```css
.toggle--active {
  /* NOTE: We need to clear `transform` since it makes the fixed overlay background (.toggle::after)
     relative to the container's boundaries instead of the viewport. For more info, see
     https://achrafkassioui.com/blog/position-fixed-and-CSS-transforms/ */
  transform: none;
  visibility: visible;
}
```

### Actual output CSS

```css
.toggle--active{visibility:visible}
```

### Expected output CSS

```css
.toggle--active{transform:none;visibility:visible}
```

**Comment by jakubpawlowicz** [comment]
Fixed alongside #1224.

**Comment by jakubpawlowicz** [comment]
The fix is out in 5.3.2.
